src/Uno.Toolkit.RuntimeTests/Tests/ResponsiveViewTests.cs

```diff
@@ -22,183 +22,162 @@ internal class ResponsiveViewTests
-						</DataTemplate>
-					</utu:ResponsiveView.NormalTemplate>
-					<utu:ResponsiveView.WideTemplate>
-						<DataTemplate>
-							<TextBlock Text="Wide" />
-						</DataTemplate>
-					</utu:ResponsiveView.WideTemplate>
-				</utu:ResponsiveView>
-			""");
-
-			await UnitTestUIContentHelperEx.SetContentAndWait(host);
-
-			Assert.AreEqual(typeof(Rectangle), host.Content.GetType());
-		}
+		var host = XamlHelper.LoadXaml<ResponsiveView>("""
+			<utu:ResponsiveView>
+				<utu:ResponsiveView.NarrowTemplate>
+					<DataTemplate>
+						<TextBlock Text="Narrow" />
+					</DataTemplate>
+				</utu:ResponsiveView.NarrowTemplate>
+				<utu:ResponsiveView.NormalTemplate>
+					<DataTemplate>
+						<Rectangle Width="400" Height="400" />
+					</DataTemplate>
+				</utu:ResponsiveView.NormalTemplate>
+				<utu:ResponsiveView.WideTemplate>
+					<DataTemplate>
+						<TextBlock Text="Wide" />
+					</DataTemplate>
+				</utu:ResponsiveView.WideTemplate>
+			</utu:ResponsiveView>
+		""");
+		await UnitTestUIContentHelperEx.SetContentAndWait(host);
+
+		host.ForceResponsiveSize(new Size(600, 400));
+		Assert.AreEqual(typeof(Rectangle), host.Content?.GetType());
 	}
 
 	[TestMethod]
 	public async Task ResponsiveView_NormalContent_ResponsiveLayout()
 	{
-		using (ResponsiveHelper.UsingDebuggableInstance())
-		{
-			ResponsiveHelper.SetDebugSize(new Size(322, 400));
-
-			var host = XamlHelper.LoadXaml<ResponsiveView>("""
-				<utu:ResponsiveView>
-					<utu:ResponsiveView.ResponsiveLayout>
-						<utu:ResponsiveLayout>
-							<utu:ResponsiveLayout.Narrowest>350</utu:ResponsiveLayout.Narrowest>
-							<utu:ResponsiveLayout.Narrow>450</utu:ResponsiveLayout.Narrow>
-							<utu:ResponsiveLayout.Normal>800</utu:ResponsiveLayout.Normal>
-							<utu:ResponsiveLayout.Wide>1200</utu:ResponsiveLayout.Wide>
-							<utu:ResponsiveLayout.Widest>1500</utu:ResponsiveLayout.Widest>
-						</utu:ResponsiveLayout>
-					</utu:ResponsiveView.ResponsiveLayout>
-					<utu:ResponsiveView.NarrowestTemplate>
-						<DataTemplate>
-							<Ellipse Width="100" Height="100" />
-						</DataTemplate>
-					</utu:ResponsiveView.NarrowestTemplate>
-					<utu:ResponsiveView.NarrowTemplate>
-						<DataTemplate>
-							<Rectangle Width="100" Height="100" />
-						</DataTemplate>
-					</utu:ResponsiveView.NarrowTemplate>
-					<utu:ResponsiveView.WideTemplate>
-						<DataTemplate>
-							<TextBlock Text="Wide" />
-						</DataTemplate>
-					</utu:ResponsiveView.WideTemplate>
-				</utu:ResponsiveView>
-			""");
-
-			await UnitTestUIContentHelperEx.SetContentAndWait(host);
-
-			Assert.AreEqual(typeof(Ellipse), host.Content.GetType());
-		}
+		var host = XamlHelper.LoadXaml<ResponsiveView>("""
+			<utu:ResponsiveView>
+				<!--
+				<utu:ResponsiveView.ResponsiveLayout>
+					<utu:ResponsiveLayout Narrowest="350"
+										  Narrow="450"
+										  Normal="800"
+										  Wide="1200"
+										  Widest="1500" />
+				</utu:ResponsiveView.ResponsiveLayout>
+				-->
```

Comment:
Code commented here still wanted ?

Reply:
i explained a couple lines below, why it was commented out:
```cs
// somehow neither attribute or member syntax work on windows...
// The attachable property 'Narrowest' was not found in type 'ResponsiveLayout'. [Line: 4 Position: 6]'
```
yes, this is the original that doesnt work, but supposed to be